json-format.md

```diff
@@ -141,17 +141,18 @@ in [Section 2.3.](#23-mapping-any-typed-attributes), with two additional
 rules:
 
 First, if an implementation determines that the type of the `data` attribute is
-`Binary` or `String`, it MUST inspect the `contenttype` attribute to determine
-whether it is indicated that the data value contains JSON data.
-
-If the `contenttype` value is either ["application/json"][RFC4627] or any media type
-with a [structured +json suffix][RFC6839], the implementation MUST translate
-the `data` attribute value into a [JSON value][JSON-Value], and set the `data`
-attribute of the envelope JSON object to this JSON value.
-
-If the `contenttype` value does not follow the [structured +json suffix][RFC6839]
-but is known to use JSON encoding, the implementation MUST translate the `data` attribute
-value into a [JSON value][JSON-Value], and set the `data` attribute of the envelope
+`Binary` or `String`, it MUST inspect the `datacontenttype` attribute to
+determine whether it is indicated that the data value contains JSON data.
+
+If the `datacontenttype` value is either ["application/json"][RFC4627] or any
+media type with a [structured +json suffix][RFC6839], the implementation MUST
+translate the `data` attribute value into a [JSON value][JSON-Value], and set
+the `data` attribute of the envelope JSON object to this JSON value.
+
+If the `datacontenttype` value does not follow the
+[structured +json suffix][RFC6839] but is known to use JSON encoding, the
+implementation MUST translate the `data` attribute value into a
+[JSON value][JSON-Value], and set the `data` attribute of the envelope
```

Comment:
Just the name change here... but things go pushed past 80 columns so I had to wrap stuff earlier.